aidies-project/client/src/components/LogoutButton.js

```diff
@@ -16,7 +17,7 @@ const LogoutButton = () => {
         return <Redirect to="/login" />
     }
     return (
-        <a href='' onClick={handleLogout}>Logout</a>
+        <a href='' onClick={handleLogout}><Button>LOGOUT</Button></a>
```

Comment:
You can set the `onClick` handler on the Material UI `<Button>` component too!
```suggestion
        <Button onClick={handleLogout}>LOGOUT</Button>
```

Reply:
Nice!